shortstories/views.py

```diff
@@ -9,3 +12,14 @@ def index(request):
 def detail(request, shortstory_id):
     shortstory =  get_object_or_404(ShortStory, pk=shortstory_id)
     return render(request,'shortstories/detail.html',{'shortstory':shortstory})
+
+def create(request):
+    if request.method == "POST":
+       form = ShortStoryForm(request.POST)
+       if form.is_valid():
+            post = form.save(commit=False)
```

Comment:
copy paste detection, why did you call your object `post`? it's a simple `shortstory`